### PR TITLE
qx.ui.layout.VBox: Local variable gaps is in the wrong scope.

### PR DESCRIPTION
**Pull request by lucapivato**

### CODE REVIEW

**Review by cboulanger — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

File: framework/source/class/qx/ui/layout/VBox.js [hunk 1]
```diff
@@ -315,12 +315,12 @@ qx.Class.define("qx.ui.layout.VBox",
 
 
       // Compute gaps
-      var spacing = this.getSpacing();
+      var spacing = this.getSpacing(), gaps;
```
Comment (oetiker) [inline framework/source/class/qx/ui/layout/VBox.js:318]:
```suggestion
      var spacing = this.getSpacing();
      var gaps;
```

Seems odd to define it on the same line as spacing

File: framework/source/class/qx/ui/layout/VBox.js [hunk 2]
```diff
@@ -521,12 +521,12 @@ qx.Class.define("qx.ui.layout.VBox",
       minHeight += percentMinHeight;
 
       // Respect gaps
-      var spacing = this.getSpacing();
+      var spacing = this.getSpacing(), gaps;
```
Comment (oetiker) [inline framework/source/class/qx/ui/layout/VBox.js:524]:
```suggestion
      var spacing = this.getSpacing();
      var gaps;
```

**Review by oetiker — APPROVED** [review]

**Review by level420 — APPROVED** [review]

File: framework/source/class/qx/ui/layout/VBox.js (hunk 2 above)
Comment (level420) [reply framework/source/class/qx/ui/layout/VBox.js:524]: @lucapivato : yes please add this change and put `var gaps;` on a separate line.

**Review by hkollmann — CHANGES_REQUESTED** [review]
see @oetiker comment! Thanks!

**Comment by level420** [comment]
@lucapivato sorry. I've highjacked your PR and changed the declaration of gaps to a separate line.

**Review by hkollmann — APPROVED** [review]

**Pull request merged**